conf/webknossos.latest.routes

```diff
@@ -63,8 +63,8 @@ GET           /users/:id/annotations
 GET           /teams                                                                            controllers.TeamController.list(isEditable: Option[Boolean])
 POST          /teams                                                                            controllers.TeamController.create
 DELETE        /teams/:id                                                                        controllers.TeamController.delete(id: String)
-GET           /teams/:id/openTasksOverview                                                      controllers.ReportController.openTasksOverview(id: String)
-GET           /teams/:id/progressOverview                                                       controllers.ReportController.projectProgressOverview(id: String)
+GET           /teams/:id/availableTasksReport                                                   controllers.ReportController.availableTasksReport(id: String)
```

Comment:
what is the difference pending pending and available? I'm asking because open was renamed to pending in other places.

Reply:
Fair question! We decided that this spot is a different perspective (which tasks are available for assignment for different users, versus how many tasks of a project are still pending).

I would also have been ok with using “pending” in both places, but we went for available in the report, compare discussion at https://scm.slack.com/archives/C5AKLAV0B/p1688653296691219?thread_ts=1688649830.633089&cid=C5AKLAV0B

Reply:
Ok, thanks for the clarification. Is this pending<->available difference explained somewhere in the docs or code comments? just for the sake of completeness..